### PR TITLE
Color token clarity

### PR DESCRIPTION
**Pull request by amazingphilippe**
# Summary | Résumé
This PR aims to simplify our color current palette and make it more predictable, as well as allowing for different color tokens to be added in the future. 

This should not dramatically change the look and field, but it will make the design and dev experience clearer.

This PR touches on grays, blues and reds only. Although the same could be done with greens and yellows. 

## Grays
- Removed 6 unused values
- Combined similar values
- Used a uniform hue across all shades
- Flipped 700 and 900 since 700 was darker than 900
- Evenly spaced out hues in a 100, 300, 700, 900 scale. 

## Reds
- Removed the flag red as it doesn't belong in the red scale. Moved it out of the numerical scale. This will help us not replace it by accident in the future. 
- Realigned based on luminosity, to match the gray scale. 
- We now have a scale with 100, 700 and 900. 

## Blues:
- Removed one unused value
- Combining the lighter blues into blue 100. 
- Focus color should not belong to the blue scale. Moved it out of the numerical scale.
- Blue 600 is used as a hover, but does not pass 7:1 on light gray background. Darkening it makes it very similar to blue 700. Therefore I'm combining them into blue 700. 
- Darkening blue 300 to pass 7:1 with white. 
- Blue 450 (header border) is very similar to blue 800. Combining into blue 800.
- Current blue 900 is essentially black, therefore we can remove it from the scale and use our existing black instead
- We can shift some colours to make the scale more coherent. The order matters: 800 -> 900, 700 -> 800, 300 -> 700. In the end we have 3 darker blue shades: 700 through 900
- Hue and saturation were adjusted to be more linear. 
- We now have a scale with 100, 700, 800 and 900

Visually explained:
image

### CODE REVIEW

File: tokens/components/lang-toggle/base.json [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
       "text": {
-        "value": "{color.blue.800.value}",
+        "value": "{color.blue.900.value}",
         "type": "color"
       }
     },
     "hover": {
       "text": {
-        "value": "{color.blue.600.value}",
+        "value": "{color.blue.800.value}",
```
Comment (melaniebmn) [inline tokens/components/lang-toggle/base.json:11]: Can we maybe use one of the lighter blues for the hover? It's very hard to see a colour difference on hover for text between blue.800 and blue.900.

**Review by melaniebmn — DISMISSED** [review]
First of all: I love the PR overview you wrote!! So detailed and I love that you also added images to explain what you did visually - thank you for that!

This is really great and a big improvement from before, thank you for taking this on. :)

I only have two comments which might be something to discuss between you and @nmakuch:
- With the new reds, the delete icon on the file uploader looks a little flat to me, can we potentially add a more vibrant red that makes it stand out a little bit more?

Screenshot 2022-12-21 at 9 52 37 AM

- For the hover of the language toggle, I find it hard to see a difference between the two colours, maybe we can use a lighter blue for the hover (if it passes contrast)?

Other than that, it looks great!

**Comment by amazingphilippe** [comment]
> add a more vibrant red that makes it stand out a little bit more?
Yes! for an icon, we can go brighter and less contrasty

**Review by ethanWallace — APPROVED** [review]
LGTM!

**Pull request merged**